app/townsquare/templates/townsquare/index.html

```diff
@@ -161,6 +161,7 @@ <h4 class="pt-3">{{type|title}} Action</h4>
 {% endblock %}
 
 {% block scripts %}
+<script src="{% static "v2/js/lib/yall.min.js" %}"></script>
```

Comment:
Can we try with native lazy loading instead? I made this with kudos and is working really nice, also with a fallback using observe
https://github.com/gitcoinco/web/pull/5940

Reply:
ok, I'll remove this :smile:

Reply:
Done! :raised_hands: